mpy-cross/mpconfigport.h

```diff
@@ -124,3 +127,33 @@ typedef long mp_off_t;
 #endif
 
 #include <stdint.h>
+
+// MSVC specifics - see windows/mpconfigport.h for explanation
+#ifdef _MSC_VER
+
+#define NORETURN                    __declspec(noreturn)
+#undef MP_NOINLINE
```

Comment:
Should be able to remove this undef now due to recent c066dadc5b91d000e2f7bba9f3e3edbf0bf33a3d

Reply:
Confirmed